src/client/firefox/commands.js

```diff
@@ -160,6 +160,10 @@ type EvaluateParam = {
 
 function evaluate(script: Script, { frameId }: EvaluateParam) {
   const params = frameId ? { frameActor: frameId } : {};
+  if (!tabTarget || !tabTarget.activeConsole) {
+    return Promise.resolve();
```

Comment:
What's the case when this happens? Is this expected until we fix some issue on the target? Perhaps we should console.warn in this case